Libplanet/HashDigest.cs

```diff
@@ -95,6 +98,14 @@ public HashDigest(ImmutableArray<byte> hashDigest)
             _byteArray = hashDigest;
         }
 
+        private HashDigest(
+            SerializationInfo info,
+            StreamingContext context)
+            : this(info?.GetValue<byte[]>("hashDigest") ??
+                throw new SerializationException("Missing the hashDigest field."))
```

Comment:
IMHO it's okay that we ignore cases with `info` is `null` here because it was called by only .NET serializer, not user codes.

```suggestion
            : this(info.GetValue<byte[]>("hashDigest"))
```

Reply:
Thanks for your suggestions. I modified them.